### DISCUSSION TITLE
Api request for orders

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#3607] [Q&A] [answered]
Hello !

I have created an API request for getting the data of the orders and I get the list of data correctly, it seems that the field event does not exist at all. I get the following list https://prnt.sc/XauIfB9oSf9I, while in the documentation as described in https://docs.pretix.eu/en/latest/api/resources/orders.html#order-resource, the field event should also be retrieved.

Thank you in advance.

**Comment by raphaelm** [comment]
As noted in the docs:

> Changed in version 2023.8: The event attribute has been added. The organizer-level endpoint has been added.

You are probably using 2023.7 if you are self-hosting, since 2023.8 has not yet been released.

**Reply by alemao8** [reply]
ok I see. Do you have an estimation of when this will be released ?

**Reply by raphaelm** [reply, marked as answer]
Yes, this month

**Reply by alemao8** [reply]
thanx
